stdlib/public/SwiftShims/RefCount.h

```diff
@@ -370,32 +386,45 @@ class RefCountBitsT {
 
   LLVM_ATTRIBUTE_ALWAYS_INLINE
   bool isImmortal() const {
-    return bool(getField(IsImmortal));
```

Comment:
Hm here's an interesting possibility… if there's a situation where the immortal bit was being set but the slow rc bit *isn't* set, then this would be a behavior change